packages/sdk/src/realtime/client.ts

```diff
@@ -45,12 +66,19 @@ export type RealTimeClient = {
   on: <K extends keyof Events>(event: K, listener: (data: Events[K]) => void) => void;
   off: <K extends keyof Events>(event: K, listener: (data: Events[K]) => void) => void;
   sessionId: string;
+  // Avatar-live audio methods (only available when model is avatar-live)
+  playAudio?: (audio: Blob | File | ArrayBuffer) => Promise<void>;
+  stopAudio?: () => void;
+  isPlayingAudio?: () => boolean;
```

Comment:
Do we really need to expose all of these?
I get `playAudio`, but the others are needed?

I think it worths adding a concrete example in `examples/sdk-core`, or even a full template, to show the actual usage.